### ISSUE TITLE
Update value4value page with new options

### ISSUE THREAD

**Issue by secondl1ght** [getAlby/website#78]
There have been some new options to use Alby for Value4Value right? Do we have a list of all the options available? I saw some talk about GitHub and Reddit and others. We should add the new options to this page.

**Comment by bumi** [comment]
yes, good point: 

currently we have: https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/content-script/batteries/index.ts#L13-L20

Reddit is upcoming. 
GitHub should also be added. 

And I hope we soon also get Medium

**Comment by secondl1ght** [comment]
Who is making those thumbnail images, @MoritzKa would you be able to make new ones for the new options?

Once I have the thumbnails I could add the copy and sections to the website. We may want to reconsider the design for this at some point - if we start to have say like 10+ options then it might not make sense to scroll through all of them on the page?

**Comment by bumi** [comment]
yes. it will not scale... it mostly is the same content but different websites.

**Comment by MoritzKa** [comment]
Maybe @auwalrg8 can help.
Thumbnails for Twitter, YouTube, etc. are [here](https://www.figma.com/file/xEPJFGFdc3HD0JS1ax2IpH/Extension-(MVP)?node-id=3517%3A22615)

**Comment by auwalrg8** [comment]
Yes I will Design the thumbnails

**Comment by secondl1ght** [comment]
Awesome thanks @auwalrg8 when those I ready I can create the new sections.

**Comment by auwalrg8** [comment]
Check out the design for the Reddit thumbnail.
I was wondering were to place the extension tab because it was hover over where the LN address supposed to be on the profile section. Let me know what you think.

https://www.figma.com/file/xEPJFGFdc3HD0JS1ax2IpH/Extension-(MVP)?node-id=3527%3A20319

**Comment by reneaaron** [comment]
I'll try to summarize what I could find out comparing code & designs. 

**Site** | **Supported by Alby** | **Integrated into Value4Value design** | **Comment**
-- | -- | -- | --
Github | ✅ | ❌ | Only supports .lnaddress in your personal repo right now
Peertube | ✅ | ✅ | bitcointv.com bitcast.online
Podcastindex | ✅ | ❌ |  
Reddit | ✅ | ✅ |  
Twitter | ✅ | ✅ |  
Vimeo | ✅ | ❌ |  
Youtube | ✅ | ✅ | Channel or video description
Medium | 🤞 soon | ❌ |  
Own website | ✅ | ✅ |

**Comment by secondl1ght** [comment]
Awesome chart @reneaaron, can we get the assets for all of these? We might also want to rethink the overall design of the Value4Value page now that we have so many options. It might not be good UX to scroll through so many option vertically.

**Comment by bumi** [comment]
@reneaaron thanks for the overview! 
Podcastindex is there but not fully supported right now. main issue there is now that most podcast have more than one recipient but we currently only allow once recipient. We need to think about the UX there.

**Comment by reneaaron** [comment]
I just updated the table. 
 - Added comment from @bumi about PeerTube
 - GitHub Repository support has been implemented, but never was used. 
 - GitHub profile support is coming with. https://github.com/getAlby/lightning-browser-extension/issues/766

**Comment by reneaaron** [comment]
I just updated the table. Let me know if I missed anything!

**Site** | **Supported by Alby** | **Integrated into Value4Value design** | **Comment**
-- | -- | -- | --
Github | ✅ | ✅ | 
Peertube | ✅ | ✅ | bitcointv.com bitcast.online
Podcastindex | ✅ | ❌ | [Limited support](https://github.com/getAlby/website/issues/78#issuecomment-1076723013)
Reddit | ✅ | ✅ |  
Twitter | ✅ | ✅ |  
Vimeo | ✅ | ❌ |  
Youtube | ✅ | ✅ | Channel or video description
Medium | ✅ | ✅ |  
Own website | ✅ | ✅ |  
Stackoverflow | ✅ | ❌
Instagram | soon 🤞 | 
Twitch | soon 🤞 |

**Comment by secondl1ght** [comment]
> another idea to add to [getalby.com/value4value](https://getalby.com/value4value) is discussed here: #90
> 
> a draft of another update is available [here](https://www.figma.com/file/xEPJFGFdc3HD0JS1ax2IpH/Extension-(MVP)?node-id=3353%3A18874) ![image](https://user-images.githubusercontent.com/13707997/176758227-902aa2cf-7bbd-4b4d-adf9-baa62f5aa25b.png) ![image](https://user-images.githubusercontent.com/13707997/176758266-4d4e88eb-2820-4354-ac8f-b935f99ee973.png)
> 
> this design becomes not usable anymore if we add even more places (e.g. Instagram or Stackoverflow)

Should I update the page to match the updated design or should we wait until the re-design is complete?

**Comment by reneaaron** [comment]
Updated the list: 

**Site** | **Supported by Alby** | **Integrated into Value4Value design** | **Comment**
-- | -- | -- | --
Geyser | ✅ | ❌ |
Github | ✅ | ✅ | 
LinkTree | ✅ | ❌ |
Medium | ✅ | ❌ |
Mixcloud | ✅ | ❌ |
Peertube | ✅ | ✅ | bitcointv.com bitcast.online
Reddit | ✅ | ❌ |
SoundCloud | ✅ | ❌ |
StackOverflow | ✅ | ❌ |
Twitter | ✅ | ✅ |  
Vida | ✅ | ❌ |
Vimeo | ✅ | ❌ |
SoundCloud | ✅ | ❌ |
Own website | ✅ | ✅ |  
Instagram | soon 🤞 | 
Twitch | soon 🤞 |

**Comment by rottingcleaner** [comment]
> Let´s wait with the next steps until we have an expert opinion by @dvoroneca.

Made a decision? It would by nice to communicate all options which Alby supports for now. GitHub is my fav.
